pages/borrow/index.tsx

```diff
@@ -285,10 +278,12 @@ const Markets: NextPage = () => {
                 marketObject =>
                   marketObject.market.address.toString() === collection.id
               );
+
               const honeyUser = collection.marketData[0].user;
               const honeyMarket = collection.marketData[0].market;
               const honeyClient = collection.marketData[0].client;
               const parsedReserves = collection.marketData[0].reserves[0].data;
+              const mData = collection.marketData[0].reserves[0];
```

Comment:
consider renaming mData to something more self explanatory/obvious (keeping 3rd parties in mind)

Reply:
nice to have